src/main/kotlin/com/terraformation/backend/accelerator/api/ParticipantProjectSpeciesController.kt

```diff
@@ -106,3 +125,7 @@ data class ParticipantProjectSpeciesPayload(
 data class GetParticipantProjectSpeciesResponsePayload(
     val participantProjectSpecies: ParticipantProjectSpeciesPayload,
 ) : SuccessResponsePayload
+
+data class UpdateParticipantProjectSpeciesPayload(
+    val participantProjectSpecies: ParticipantProjectSpeciesPayload,
```

Comment:
It'd be cleaner to put the relevant data fields directly in this payload. Otherwise callers have to populate all three IDs in the inner payload (since they're non-nullable) even though the participant project species ID is already part of the URL path.